### ISSUE TITLE
Allow marker to persist when entering the editor

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#9556]
### Description

Here we are about to push the Edit button seen in the image.
All our attention is focused on the marker you see here.
### Screenshots

![20230320T133709](https://user-images.githubusercontent.com/1230959/226255476-0c5f55ff-5f46-4d8d-91a3-11cc4c664881.jpg)

Well, alas, when we enter iD it disappears.
Some users would "pay cash" if you could kindly keep the reference marker on the screen after entering iD.
It is only a harmless reference marker. It is not some tagged map data etc.

Yes, it is where grandma dropped us off right across the street from the buried treasure. Not important. It's the user's own business what it is.

**Comment by 1ec5** [comment]
More generally, iD could support a [hash parameter or query parameter](https://github.com/openstreetmap/iD/blob/c01adb01fabe0dfdc4d9e5b88d15967f56a20859/API.md#id-standalone) indicating a coordinate or bbox to mark using a synthesized data layer. The Tasking Manager currently relies on the `gpx` parameter, passing in a specially generated GPX file that contains a five-coordinate trace, but this feels like a hack. Meanwhile, MapRoulette tasks can’t mark the task’s geometry at all in iD, whereas they can add reference layers to JOSM. Both platforms would benefit from being able to pass in the coordinates more straightforwardly.

Persisting osm.org’s marker would similarly require this feature, but it would also require openstreetmap-website to include the marker coordinates in the edit URL, whereas it currently drops the coordinates. That’s outside the scope of this repository.

/ref #5856 https://github.com/maproulette/maproulette3/issues/1802#issuecomment-1225510497

**Comment by tyrasd** [comment]
I agree that this could be useful for applications such as QA tools that want to point to a particular location with a data issue.

From a UI/UX perspective, the question would be how such a marker is implemented: e.g. does it persist for the full edit session, or does it vanish once one clicks on a different feature?
